dist-karaf/apache-brooklyn/src/main/resources/etc/default.catalog.bom

```diff
@@ -0,0 +1,371 @@
+
+# this catalog bom is an illustration supplying a few useful sample items
```

Comment:
I had a go at this and it turned out to be not entirely trivial - I think it would be worth doing this as a separate task, and not holding up this PR for it.